src/FixedDataTable.js

```diff
@@ -751,8 +764,15 @@ var FixedDataTable = createReactClass({
     /*number|string*/ columnKey,
     /*object*/ event
   ) {
+
+    var nowMs = Date.now();
+    if (this.state.lastColumnResizeStart && nowMs - this.state.lastColumnResizeStart < DOUBLE_CLICK_TIMEOUT) {
+      (this.props.onColumnResizeDoubleClick || emptyFunction)(columnKey);
+    }
```

Comment:
On a double click you were getting 2 regular resize callbacks. I wasn't trying to "reinvent the wheel" I just thought it easier to handle it all at this point.